stdlib/float32.gr

```diff
@@ -1,3 +1,7 @@
+/**
+ * @module Float32: Utilities for working with the Float32 type.
+ * @example import Float32 from "float32"
```

Comment:
Crud, I just realized that we didn't put a `@since` attribute on the Float64 module definition. Can you add it here and probably open another PR for adding it to Float64?

Reply:
Sure